examples/direct_fidelity_estimation.py

```diff
@@ -30,16 +30,29 @@
 
 
 def build_circuit():
```

Comment:
Add return type annotation to `build_circuit()`

Reply:
Done.